utility_code/lib_py3/plugin_data.py

```diff
@@ -286,3 +294,74 @@ def pos(self):
         elif self.parent is not None:
             return self.parent.pos
         return None
+
+
+class MonumentaVanity(NbtPathDebug):
```

Comment:
Using NbtPathDebug here will fail, as this does not provide the expected values like `self.nbt` being an NBT tag. Similarly, I'm not sure if `parent` will be helpful without a new debug class that can handle json data.

Reply:
Expanding on that, each NbtPathDebug object can report a string describing how to get to this object for use in stack traces by following the parent object back to wherever it came from. Those methods won't work for non-NBT data.

Reply:
I copied this from graves, and those don't have NBT either. So I think this should work? If that just works by chance though when the graves classes should be changed as well I think (but to what?)